prometheus_hardware_exporter/collector.py

```diff
@@ -5,13 +5,134 @@
 
 from prometheus_client.metrics_core import GaugeMetricFamily, InfoMetricFamily
 
+from .collectors.perccli import PercCLI
 from .collectors.sasircu import LSISASCollectorHelper, Sasircu
 from .collectors.storcli import StorCLI
 from .core import BlockingCollector, Payload, Specification
 
 logger = getLogger(__name__)
 
 
+class PowerEdgeRAIDCollector(BlockingCollector):
+    """Collector for PowerEdge RAID controller."""
+
+    perccli = PercCLI()
+
+    @property
+    def specifications(self) -> List[Specification]:
+        return [
+            Specification(
+                name="perccli_command_success",
+                documentation="Indicates if the command is successful or not",
+                metric_class=GaugeMetricFamily,
+            ),
+            Specification(
+                name="perccli_command_ctrl_success",
+                documentation="Indicates if the command for each controller is successful or not",
+                labels=["controller_id"],
+                metric_class=GaugeMetricFamily,
+            ),
+            Specification(
+                name="poweredgeraid_controller_exists",
```

Comment:
1. 
What's different between poweredgeraid_controllers and poweredgeraid_controller_exists ?

exists is true/false,  which I think it's more likely an alert rule , not a metrics. I prefer to avoid boolean metrics unless it is neccessary. 

2. this metrics is not available in https://docs.google.com/spreadsheets/d/1KiQ5cS36zfjf8V32RoD1RKx9q01AhgoKvBQRkonQv5o/edit#gid=1149993398